docs/user-guide/ui.md

```diff
@@ -2,48 +2,82 @@
 ---
 
 
-Kubernetes has a web-based user interface that allows you to deploy containerized
-applications to a Kubernetes cluster, troubleshoot them, and manage the cluster itself.
+The web-based user interface of Kubernetes (Dashboard) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon sets, Pet sets, Replica sets, Jobs, Replication controllers and corresponding Services, or Pods.

```

Comment:
I think the important word here is 'Dashboard'. Its description should be what is in brackets: 
Dashboard (the web-based user interface of Kubernetes) allows...
It is possible to misinterpret the current form if you don't already know what you're looking at.